.github/workflows/governance.yml

```
@@ -0,0 +1,308 @@
+        run: |
+          set -euo pipefail
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
+          echo "Changed files:"
+          echo "$CHANGED_FILES"
+          echo ""
+
+          # If rules are unsigned, fall back to a minimal bootstrap set.
+          SIGNATURE=$(jq -r '.signature.signature // empty' rules.json)
+          if [[ -z "$SIGNATURE" ]]; then
+            PROTECTED_PATTERNS=$(cat <<'EOF'
+.github/workflows/**
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)